src/game_renderer/RendererSetup.cpp

```diff
@@ -65,10 +21,15 @@ GLFWwindow* RendererSetup::InitOpenGL() {
   glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 3);
   glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 2);
 
+  GLFWwindow* window;
+#ifdef DEBUG
+  window =
+      glfwCreateWindow(WINDOW_WIDTH, WINDOW_HEIGHT, "Level Editor", NULL, NULL);
```

Comment:
Not that it matters, but I don't think we want our main game's window to be titled Level Editor